easybuild/easyconfigs/s/scikit-image/scikit-image-0.21.0-foss-2022b.eb

```diff
@@ -0,0 +1,48 @@
+easyblock = 'PythonBundle'
+
+name = 'scikit-image'
+version = '0.21.0'
+
+homepage = 'https://scikit-image.org/'
+description = "scikit-image is a collection of algorithms for image processing."
+
+toolchain = {'name': 'foss', 'version': '2022b'}
+
+builddependencies = [
+    ('meson-python', '0.11.0'),
+]
+
+dependencies = [
+    ('Python', '3.10.8'),
+    ('matplotlib', '3.7.0'),
+    ('Pillow', '9.4.0'),
+    ('networkx', '3.0'),
+    ('dask', '2023.7.1'),
+    ('imageio', '2.31.1'),
+]
+
+use_pip = True
+sanity_pip_check = True
+
+exts_list = [
+    ('PyWavelets', '1.4.1', {
+        'modulename': 'pywt',
+        'checksums': ['6437af3ddf083118c26d8f97ab43b0724b956c9f958e9ea788659f6a2834ba93'],
+    }),
+    ('imread', '0.7.4', {
+        'checksums': ['0487adef11a22168700968c1727020361a72f6132b6ced2b8826b02d8cbf744f'],
+    }),
+    ('tifffile', '2023.7.18', {
+        'checksums': ['5a5a624b2f7ab7f37e9ec4174ae2df1805b9658f89013f9b4b5550672f65f2a1'],
+    }),
+    ('lazy_loader', '0.3', {
+        'checksums': ['3b68898e34f5b2a29daaaac172c6555512d0f32074f147e2254e4a6d9d838f37'],
+    }),
+    (name, version, {
+        'modulename': 'skimage',
+        'source_tmpl': 'scikit_image-%(version)s.tar.gz',
+        'checksums': ['b33e823c54e6f11873ea390ee49ef832b82b9f70752c8759efd09d5a4e3d87f0'],
+    }),
+]
```

Comment:
Similar question here: `pooch` has disappeared since prior version. Also intentional I suppose?

Reply:
[`pooch`](https://github.com/easybuilders/easybuild-easyconfigs/blob/develop/easybuild/easyconfigs/p/Python/Python-3.10.8-GCCcore-12.2.0.eb#L492) is in the Python easyconfig.

Reply:
Clear!